examples/ConfigActuation/update_config.py

```diff
@@ -0,0 +1,138 @@
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# The views and conclusions contained in the software and documentation
+# are those of the authors and should not be interpreted as representing
+# official policies, either expressed or implied, of the FreeBSD
+# Project.
+#
+# This material was prepared as an account of work sponsored by an
+# agency of the United States Government.  Neither the United States
+# Government nor the United States Department of Energy, nor Battelle,
+# nor any of their employees, nor any jurisdiction or organization that
+# has cooperated in the development of these materials, makes any
+# warranty, express or implied, or assumes any legal liability or
+# responsibility for the accuracy, completeness, or usefulness or any
+# information, apparatus, product, software, or process disclosed, or
+# represents that its use would not infringe privately owned rights.
+#
+# Reference herein to any specific commercial product, process, or
+# service by trade name, trademark, manufacturer, or otherwise does not
+# necessarily constitute or imply its endorsement, recommendation, or
+# favoring by the United States Government or any agency thereof, or
+# Battelle Memorial Institute. The views and opinions of authors
+# expressed herein do not necessarily state or reflect those of the
+# United States Government or any agency thereof.
+#
+# PACIFIC NORTHWEST NATIONAL LABORATORY
+# operated by BATTELLE for the UNITED STATES DEPARTMENT OF ENERGY
+# under Contract DE-AC05-76RL01830
+
+# }}}
+
+"""Script for adding or updating a JSON file in an agent's
+:ref:`configuration store <VOLTTRON-Configuration-Store>`.
+
+If the file does not exist it will be created.
+"""
+
+import argparse
+import json
+
+from volttron.platform import get_address
+from volttron.platform.agent.known_identities import CONFIGURATION_STORE
+from volttron.platform.keystore import KeyStore, KnownHostsStore
+from volttron.platform.vip.agent.utils import build_agent
+
+def get_keys():
+    """Gets keys from keystore and known-hosts store
+
+    :returns: Keys for connecting to the platform
+    :rtype: dict
+    """
+    hosts = KnownHostsStore()
+    serverkey = hosts.serverkey(get_address())
+    key_store = KeyStore()
+    publickey = key_store.public
+    secretkey = key_store.secret
+    return {'publickey': publickey,
+            'secretkey': secretkey,
+            'serverkey': serverkey}
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('vip_identity',
+                        help='VIP Identity whose configstore we want to access')
+    parser.add_argument('filename',
+                        help='Name of the configstore file to edit')
+    parser.add_argument('key',
+                        help='Key to add or edit in the config')
+    parser.add_argument('value',
+                        help='Value that the key will assume')
```

Comment:
Value that the key will assume...is this a float, int string double? json string?  Why does it have to be a float?